docs/_layouts/global.html

```diff
@@ -207,14 +205,79 @@ <h1 class="title">{{ page.title }}</h1>
     })
 </script>
 
+<!-- Pure JavaScript implementation of ScrollSpy -->
```

Comment:
The code is from scratch, but it emulates a jQuery plugin called ScrollSpy that tracks where the user is on the page